### ISSUE TITLE
Thermostat - Current Temperature not being refreshed

### ISSUE THREAD

**Issue by alisdairjsmyth** [pdlove/homebridge-smartthings#35]
The current temperature is not being updated as part of the polling process.  I am presently watching the temperature as reported in Smartthings decline, and the temperature in the HomeKit App remain constant.

Environment:
- Homebridge-Smartthings: 0.3.5
- Homebridge-Smartthings SmartApp: 0.3.6
- HomeKit App: Eve
- `polling_seconds`: Not defined and set in config.json

**Comment by alisdairjsmyth** [comment]
I have also tried setting `polling_seconds` - current temperature does not update in line with defined frequency

**Comment by pdlove** [comment]
Are you just sitting there staring at the screen? I'm seeing EVE only refresh numerical values if you back up and go back into the screen. I'll verify another plugin works this way.

**Comment by alisdairjsmyth** [comment]
> Are you just sitting there staring at the screen?
Not quite.  I have tried going into and out the screen.  I tried looking at the temperature through "Types" and through "Rooms".  I tried asking Siri what the temperature was in the room.  And I also tried using the Insteon App.  All produced the same result - a stale value of the "current temperature".

At the moment the Thermostat is in the mode off, so the only dynamic value is "current temperature".  As a consequence I can't comment on whether any values are being refreshed.

**Comment by tonesto7** [comment]
I can verify this as well. None of the state data is updating... By that I mean open/close, temp, presence, etc

**Comment by pdlove** [comment]
Please verify this is fixed.

**Comment by alisdairjsmyth** [comment]
I am travelling at the moment so will not be able to verify for a couple
weeks.

**Comment by tannebil** [comment]
I'm running the latest code and I'm seeing the same problem with both temperature and contact on a SmartSense Multi and an Aeon contact sensor. Both are working fine in the SmartThings app.

**Comment by tannebil** [comment]
I updated and changed the polling interval to 60. All the contact and temperature sensors are updating. Thanks!

**Comment by tannebil** [comment]
Spoke too soon. While my SmartThings lights and Aeon contact sensor appear to be fine, the SmartSense Multi has disappeared from HomeKit. Still functioning fine in SmartThings. I deleted it in the JSON Complete API and added it back. After doing it a few times, the log showed it added although it did not how in HomeKit. I restarted homebridge, it initialized with the rest of the SmartThings devices, and now it's back in HomeKit. Hopefully just a one-time glitch.
